app/main.py

```diff
@@ -2,0 +1,17 @@
+def copy_file(command: str) -> None:
+    command_parts = command.split()
+    if (
+        len(command_parts) != 3
+        or command_parts[1] == command_parts[2]
+        or command_parts[0] != "cp"
+    ):
+        return
+
+    with open(command_parts[1], "r") as current_file, open(
```

Comment:
The variable names `current_file` and `new_file` are not very descriptive. Consider using more meaningful names, like `source_file` and `destination_file`.